src/lib/components/workflow/filter-search/close-filter-button.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { getContext } from 'svelte';
+  
+  import Button from '$lib/holocene/button.svelte';
+  import { translate } from '$lib/i18n/translate';
+  
+  import { FILTER_CONTEXT, type FilterContext } from './index.svelte';
+
+  const { resetFilter } = getContext<FilterContext>(FILTER_CONTEXT);
+</script>
+
+<Button
+  aria-label={translate('close')}
+  borderRadiusModifier="square-left"
+  leadingIcon="chevron-left"
+  on:click={resetFilter}
+  style="padding: 0"
```

Comment:
Wondering how we feel about overriding css like this? Or do we want to add something to support this in the `Button` component?

Reply:
Oh I see this is for that "collapse" button we were talking about the other day. Personally I could do without it, or maybe just an `<IconButton />` with an `X` icon next to the filters.

Reply:
So the default `<IconButton />` looks a little wonky here

![Screenshot 2023-09-06 at 12 39 28 PM](https://github.com/temporalio/ui/assets/15069288/c4507b1d-15ed-44ff-8af2-fe5aa797a004)

Let's discuss this a bit more and maybe get design's input.

Reply:
[Updated](https://github.com/temporalio/ui/pull/1613/commits/2e3b17f83eba67067ed0dcc924685f8a1c8522c9) to use `<IconButton />`  ✅